allennlp/training/metrics/span_based_f1_measure.py

```diff
@@ -69,19 +70,30 @@ def __call__(self,
             shape as the ``predictions`` tensor without the ``num_classes`` dimension.
         mask: ``torch.Tensor``, optional (default = None).
             A masking tensor the same size as ``gold_labels``.
+        prediction_map: ``torch.Tensor``, optional (default = None).
```

Comment:
Can you add little bit of context about the case in which this is required (i.e when the model produces distributions over different tags depending on the frame), just because it's hard to imagine why this is necessary without this context.